deltachat-ios/AppDelegate.swift

```diff
@@ -64,6 +73,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         dcAccounts.openDatabase()
         migrateToDcAccounts()
 
+        self.launchOptions = launchOptions
+        finishAppInitialization()
+        return true
+    }
+
+    // finishes the app initialization which depends on the successful access to the keychain
+    func finishAppInitialization() {
```

Comment:
naming suggestion: `continueDidFinishLaunchingWithOptions`, that's what it is - and this is also nicely grep'able.